api/pom.xml

```diff
@@ -98,9 +98,9 @@
                 <configuration>
                     <specMode>jakarta</specMode>
                     <spec>
-                        <nonFinal>false</nonFinal>
+                        <nonFinal>true</nonFinal>
                         <jarType>api</jarType>
-                        <specVersion>1.1</specVersion>
+                        <specVersion>1.0</specVersion>
```

Comment:
I know that this one is already merged. Just wanted to note that I didn't fully understand back then how this plugin work. Especially the difference between `specVersion` and `newSpecVersion`.